### PR TITLE
Update dark-sites.config

### PR DESCRIPTION
**Pull request by realpixelcode**
Added several Mastodon servers with dark themes. Removed Graphene's and Tuta's websites because they're not dark by default.

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -364,6 +372,7 @@ ecosia.org
 ecys.xyz
 edit-csv.net
 editor.method.ac
+edi.social
```
Comment (Myshor) [inline src/config/dark-sites.config:375]: not sorted alphabetically

should be before edit-csv.net

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -736,6 +753,7 @@ mrdemoapple.uk
 mrms.cz
 mrquantumoff.dev
 mrtipson.github.io/otz-builds/
+mstdn.social
```
Comment (Myshor) [inline src/config/dark-sites.config:756]: not sorted alphabetically

should be under ms-paint-i.de

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -1145,6 +1168,7 @@ ultrajs.dev
 undergroundcellar.com
 underlords.com
 undertale.com
+universeodon.com
```
Comment (Myshor) [inline src/config/dark-sites.config:1171]: not sorted alphabetically

should be under universe.eveonline.com

**Review by Myshor — CHANGES_REQUESTED** [review]
Fix sorting.

File: src/config/dark-sites.config [hunk 4]
```diff
@@ -82,6 +84,7 @@ archive.ragtag.moe
 archon.gg
 arcturusmc.xyz
 ardov.me
+ard.social
```
[the remark this answers is not in the document]
Comment (realpixelcode) [reply src/config/dark-sites.config:86]: > . is above the letters so it should be above ardov.me

thx, I didn't know that

File: src/config/dark-sites.config [hunk 5]
```diff
@@ -176,6 +180,7 @@ brodierobertson.xyz
 browserslist.vercel.app
 brunotome.dev
 bsodium.fr
+bonn.social
```
[the remark this answers is not in the document]
Comment (realpixelcode) [reply src/config/dark-sites.config]: lol, I'm stupid

**Review by Myshor — APPROVED** [review]

**Pull request merged**